### PR TITLE
[MOV] l10n_fr: Rename into l10n_fr_account

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_fr/__manifest__.py [hunk 1]
```diff
@@ -1,51 +1,22 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
-    'name': 'France - Accounting',
-    'website': 'https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/france.html',
+    'name': 'France - Localizations',
     'icon': '/account/static/description/l10n.png',
     'countries': ['fr'],
     'version': '2.1',
     'category': 'Accounting/Localizations/Account Charts',
```
Comment (Bertrand2) [inline addons/l10n_fr/__manifest__.py:7]: I guess you can adapt the category as well :ok_hand:

File: addons/l10n_fr/__manifest__.py [hunk 2]
```diff
@@ -1,51 +1,22 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
-    'name': 'France - Accounting',
-    'website': 'https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/france.html',
+    'name': 'France - Localizations',
     'icon': '/account/static/description/l10n.png',
```
Comment (Bertrand2) [inline addons/l10n_fr/__manifest__.py:4]:
```suggestion
    'icon': '/base/static/description/l10n.png',
```

File: addons/l10n_fr/models/__init__.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (Bertrand2) [inline addons/l10n_fr/models/__init__.py:1]:
```suggestion
```

File: addons/l10n_fr/models/res_partner.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
```
Comment (Bertrand2) [inline addons/l10n_fr/models/res_partner.py:1]:
```suggestion
```

File: addons/l10n_fr_account/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from . import account_move
```
Comment (Bertrand2) [inline addons/l10n_fr_account/models/__init__.py:2]:
```suggestion
# Part of Odoo. See LICENSE file for full copyright and licensing details.

from . import account_move
```

File: addons/l10n_fr_account/models/res_company.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```
Comment (Bertrand2) [inline addons/l10n_fr_account/models/res_company.py:1]:
```suggestion
```

File: addons/l10n_fr/__init__.py [hunk 7]
```diff
@@ -1,27 +1,4 @@
 # -*- coding: utf-8 -*-
```
Comment (Bertrand2) [inline addons/l10n_fr/__init__.py:1]:
```suggestion
```

**Review by Bertrand2 — COMMENTED** [review]
Maybe the method `_is_accounting_unalterable` and its calls should move to `l10n_fr_account as well`?

**Comment by jco-odoo** [comment]
@tivisse Has someone checked if payroll without accounting is needed that much?

File: addons/l10n_fr/__manifest__.py (hunk 2 above)
Comment (jco-odoo) [inline addons/l10n_fr/__manifest__.py:4]: Why still keep l10n_fr?  Just define the little fields on company and partner in both modules.

File: addons/l10n_fr_account/i18n/fr.po
Comment (william-andre) [inline addons/l10n_fr_account/i18n/fr.po:1]: Translation files contain comments with the module name, I think these are necessary for the translations to work correctly

Comment (tivisse) [reply addons/l10n_fr_account/i18n/fr.po:1]: Asked to the translation team, they confirmed that those comments aren't interpreted.

**Comment by tivisse** [comment]
@robodoo rebase-ff r+

**Comment by jco-odoo** [comment]
@tivisse We explicitly asked not to have l10n_fr anymore, so we don't get half- empty modules everywhere...

**Pull request closed without merging**